assets/templateFloxEnv/manifest.toml

```diff
@@ -9,20 +9,32 @@
 
 [install]
 _FLOX_INIT_INSTALL
-# Set an environment variable.
-# These variables may not reference once another
-# and are added to the environment without expansion.
+# Set environment variables in the 'vars' section.
+# These variables may not reference once another, and are added to
+# the environment without first expanding them.
```

Comment:
It's saying that it's not expanded before putting it in the activation script, so it will have to be present at runtime in the shell that launches the environment.

Reply:
But why would you want to set a variable to something with a literal `$` in it? Not to mention I don't think `-` is even valid in a variable name. Not really from this PR so nonblocking/you don't need to address here